src/views/open-api-docs-view/utils/get-operation-props.ts

```diff
@@ -7,21 +7,17 @@
 import { snakeCase } from 'change-case'
 import slugify from 'slugify'
 // Local
-import {
-	getBodyParameterProps,
-	getPropertyDetailPropsFromParameter,
-	truncateHcpOperationPath,
-} from './'
+import { getRequestData, getResponseData, truncateHcpOperationPath } from './'
```

Comment:
I've split out `getRequestData` and `getResponseData` to separate functions to try to keep this file easy to skim.